packages/guides/src/Nodes/SectionNode.php

```diff
@@ -33,4 +46,22 @@ public function getTitles(): array
 
         return $titles;
     }
+
+    public function addAnchor(string $anchor): void
+    {
+        $this->identifier = trim($anchor);
```

Comment:
The last anchor is determining the identifier of the section?

Reply:
If no explicit anchor is given, the identifier is derived from a converted form of the title text. if one explicit anchor is given it is the identifier of the anchor directly following the section, therefore the anchor last found before the section